.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,115 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: StreamPipes Bug report
+description: Problems and issues with Apache StreamPipes
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: "
+        <p align='center'>
+        
+        <img src='https://streampipes.apache.org/img/sp-logo-color.png' alt='StreamPipes Logo' title='Apache StreamPipes Logo' width='50%' align='center'/>
+        </p>
+        
+        <br>
+        
+        Thank you for taking the time to report the issue!
+
+        We greatly appreciate the community's efforts to improve StreamPipes.
+
+        Please note that you do not need to create an issue if you want to submit a change!
+
+        You can open a <a href='https://github.com/apache/streampipes/pulls'>pull request</a> immediately instead.
+        "
+
+  - type: dropdown
+    attributes:
+      label: Apache StreamPipes version
+      description: What Apache StreamPipes version are you using?
+      multiple: false
+      options:
+        - "0.70.0"
+        - "dev (current development state)"
+        - "Other StreamPipes version (please specify below)"
+
+  - type: dropdown
+    attributes:
+      label: Affected StreamPipes components
+      description: >
+        Which StreamPipes components are affected by this bug?
+        Where did the bug occur?
+      multiple: true
+      options:
+        - Backend
```

Comment:
Somehow the mapping from these components to "core/extensions" went wrong during issue migration.
Should we use these labels anyway?
E.g., "Core", "Extensions (pipeline elements and adapters)"?

Reply:
I have already cleaned them up and corrected the "wrong" assignments.
So we can use the labels as planned.
Are you missing any other options here?
Or in general another input field?